src/amo/reducers/users.js

```diff
@@ -141,7 +143,8 @@ export const finishEditUserAccount = (): FinishEditUserAccountAction => {
 
 type EditUserAccountParams = {|
   errorHandlerId: string,
-  picture?: File | null,
+  notifications: NotificationsUpdateType,
+  picture: File | null,
```

Comment:
Why does this patch include this change? It doesn't seem to relate to notifications. Is it just an unrelated problem you found when writing this patch?

Reply:
I realized picture was always present, no need to mark it as optional.